config/ant-phase-verify.xml

```diff
@@ -221,6 +221,8 @@
           <exclude name="**/asttreestringprinter/**/*"/>
           <!-- Indentation checks use "warn" -->
           <exclude name="**/resources-noncompilable/**/checks/indentation/**/*"/>
+          <!-- BlockComment test files do not need 'ok', 'violation, or 'Config:' comments -->
+          <exclude name="**/utils/blockcommentposition/**/*"/>
```

Comment:
Adding this exclude here will allow us to remove all `**/blockcommentposition/**.*` suppressions from `checkstyle_input_suppressions.xml`.